modules/models/src/main/java/org/eclipse/xpanse/modules/models/servicetemplate/Deployment.java

```diff
@@ -37,10 +37,23 @@ public class Deployment implements Serializable {
     @UniqueElements
     @Schema(
             description =
-                    "The variables for the deployment, which will be passed to the deployer.The"
-                        + " list elements must be unique. All variables are put together to build a"
-                        + " JSON 'object' with each variable as a property of this object.")
-    private List<DeployVariable> variables;
+                    "The input variables for the deployment, which will be passed to the"
+                        + " deployer.The list elements must be unique. All input variables are put"
+                        + " together to build a JSON 'object' with each variable as a property of"
+                        + " this object.")
+    private List<InputVariable> inputVariables;
+
+    @Valid
+    @NotNull
+    @NotEmpty
+    @UniqueElements
+    @Schema(
+            description =
+                    "The output variables for the deployment, which will be generated by the"
+                        + " deployer.The list elements must be unique. All output variables are put"
+                        + " together to build a JSON 'object' with each variable as a property of"
```

Comment:
Does the last line here is needed? - 

All output variables are put"
                        + " together to build a JSON 'object' with each variable as a property of"
                        + " this object.